handlers/tweet.js

```diff
@@ -0,0 +1,330 @@
+            next();
+        } catch (error) {
+            response.sendStatus(401);
+        }
+    }
+};
+
+/**
+ * @openapi
+ * /tweets:
+ *   get:
+ *     description: Returns all tweets.
+ *     tags:
+ *      - Tweet
+ *     responses:
+ *       200:
+ *        description: Returns all tweets.
+ *        content:
+ *          application/json:
+ *             schema: 
+ *                type: object
+ *                properties:
+ *                  id:
+ *                     type: integer
+ *                  user_id:
+ *                     type: integer
+ *                  text:
+ *                     type: string
+ *                  photo:
+ *                     type: string
+ */
+tweet.get('/tweets', async (request, response) => {
+    try {
+        const tweets = await tweetController.getTweets();
+        response.json(tweets);
+    } catch (error) {
+        request.log.error(error);
+        response.end(JSON.stringify(error)).status(500);
+    }
+});
+
+
+/**
+ * @openapi
+ * /tweets/{id}/tweet:
+ *   get:
+ *     description: Returns a specific tweet.
+ *     tags:
+ *      - Tweet
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         description: tweet id
+ *         schema:
+ *          type: integer
+ *     responses:
+ *       200:
+ *        description: Returns a specific tweet.
+ *        content:
+ *          application/json:
+ *             schema: 
+ *                type: object
+ *                properties:
+ *                  id:
+ *                     type: integer
+ *                  user_id:
+ *                     type: integer
+ *                  text:
+ *                     type: string
+ *                  photo:
+ *                     type: string
+ */
+tweet.get('/tweets/:id/tweet',
+    validate([param('id').isInt({ min: 1 })]),
+    async (request, response) => {
+        try {
+            const tweet = await tweetController.getTweet(request.params.id);
+            response.json(tweet);
+        } catch (error) {
+            request.log.error(error);
+            response.end(JSON.stringify(error)).status(500);
+        }
+    });
+
+/**
+ * @openapi
+ * /tweets/{id}/tweets:
+ *   get:
+ *     description: Returns all tweets by a specific user.
+ *     tags:
+ *      - Tweet
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         description: user id
+ *         schema:
+ *          type: integer
+ *     responses:
+ *       200:
+ *        description: Returns all tweets by a specific user.
+ *        content:
+ *          application/json:
+ *             schema: 
+ *                type: object
+ *                properties:
+ *                  id:
+ *                     type: integer
+ *                  user_id:
+ *                     type: integer
+ *                  text:
+ *                     type: string
+ *                  photo:
+ *                     type: string
+ */
+tweet.get('/tweets/:id/tweets',
```

Comment:
All tweets by user should be implemented as `/users/:id/tweets`, where id is the user id. As you can see the the user resource is not represented, and tweets two times doesn't mean much.

In order to implement `/users/:id/tweets` you should move this handler to user.js handlers and import the tweetController there. In this way you could call the getTweetByUser function in the user handler